hack/undeploy-karmada.sh

```diff
@@ -46,3 +46,11 @@ kubectl delete ns karmada-system --kubeconfig="${HOST_CLUSTER_KUBECONFIG}"
 kubectl config delete-cluster karmada-apiserver --kubeconfig="${HOST_CLUSTER_KUBECONFIG}"
 kubectl config unset users.karmada-apiserver --kubeconfig="${HOST_CLUSTER_KUBECONFIG}"
 kubectl config delete-context karmada-apiserver --kubeconfig="${HOST_CLUSTER_KUBECONFIG}"
+
+ETCD_HOST_IP=$(kubectl get pod -l app=etcd -n karmada-system -o jsonpath='{.items[0].status.hostIP}')
```

Comment:
I guess the `ETCD_HOST_IP` should be obtained before the `karmada-system` was deleted.

I just test it in my environment, but a error occurred.

```
[root@master67 karmada]# hack/undeploy-karmada.sh ~/.kube/config karmada-host
Switched to context "karmada-host".
namespace "karmada-system" deleted
deleted cluster karmada-apiserver from /root/.kube/config
Property "users.karmada-apiserver" unset.
deleted context karmada-apiserver from /root/.kube/config
error: error executing jsonpath "{.items[0].status.hostIP}": Error executing template: array index out of bounds: index 0, length 0. Printing more information for debugging the template:
	template was:
		{.items[0].status.hostIP}
	object given to jsonpath engine was:
		map[string]interface {}{"apiVersion":"v1", "items":[]interface {}{}, "kind":"List", "metadata":map[string]interface {}{"resourceVersion":"", "selfLink":""}}
```